kinesis-video-gstreamer-plugin/src/gstkvssink.cpp

```diff
@@ -988,11 +979,13 @@ gst_kvs_sink_handle_sink_event (GstCollectPads *pads,
 }
 
 void create_kinesis_video_frame(Frame *frame, const nanoseconds &pts, const nanoseconds &dts, FRAME_FLAGS flags,
-                                void *frame_data, size_t len, uint64_t track_id, uint32_t index) {
+                                void *frame_data, size_t len, uint64_t track_id, uint32_t index, uint32_t timecode_scale_ms) {
+    UINT64 timecode_scale = (UINT64) timecode_scale_ms * HUNDREDS_OF_NANOS_IN_A_MILLISECOND;
```

Comment:
This can be pre-computed as this doesn't change

Reply:
removed timecode scale.